README.md

```diff
@@ -1,68 +1,51 @@
-# EVE
 
-This project aims to simulate basic lifeforms evolving and getting smarter, in a simplified manner.
+# EVE
 
-## Development
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)[![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-### Setup
+##### Simulate simple, artificial lifeforms as they learn and evolve
 
-1. Install cmake & [Catch2](https://github.com/catchorg/Catch2) on your OS
-2. Clone this repository:  
-   HTTPS: `git clone https://github.com/Xerus2000/eve`  
-   SSH: `git clone git@github.com:Xerus2000/eve.git`
-3. Build 'eve' via CLion so that all tasks become available for running
-4. In CLion go to `Settings > Editor > File and Code Templates` and remove the "Created by" header in c/cpp files
-5. Execute this to set up the git commit message hooks:  
-   `git config core.hooksPath .dev/githooks`
+----
 
-### Structure
+#### Table of Content
```

Comment:
isnt it contents?

Reply:
a plural of content doesn't really make sense, but I see that everyone else uses contents